src/app/clusters/channel-server/channel-server.cpp

```diff
@@ -197,6 +199,12 @@ CHIP_ERROR ChannelAttrAccess::ReadCurrentChannelAttribute(app::AttributeValueEnc
     return delegate->HandleGetCurrentChannel(aEncoder);
 }
 
+CHIP_ERROR ChannelAttrAccess::ReadRevisionAttribute(EndpointId endpoint, app::AttributeValueEncoder & aEncoder, Delegate * delegate)
+{
+    uint16_t clusterRevision = delegate->GetClusterRevision(endpoint);
```

Comment:
All the other code in this file null-checks the delegate.  Why not here?  This applies to a bunch of files in this PR.